pkg/controllers/consistency/events.go

```diff
@@ -17,10 +17,11 @@ package consistency
 import (
 	v1 "k8s.io/api/core/v1"
 
+	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
 	"github.com/aws/karpenter-core/pkg/events"
 )
 
-func CheckEvent(node *v1.Node, message string) []events.Event {
+func CheckEvent(node *v1.Node, machine *v1alpha5.Machine, message string) []events.Event {
```

Comment:
I'm not totally sold that we want to event on both of these. Thoughts on just eventing on the machine or just the node?

Reply:
Updated the eventing behavior so that we only event on one